### ISSUE TITLE
Can't Connect to Poolin

### ISSUE THREAD

**Issue by acos0874** [RainbowMiner/RainbowMiner#1929]
Poolin has been working fine for weeks, but suddenly today my rigs won't connect. I noticed the pool URL being used by RBM is completely different to the suggested one. Could this be the problem?

**Comment by RainbowMiner** [comment]
Yes, they have changed the stratum address back to "normal". The other stratum has been only temporary. Also, they changed the API regarding the hashrate. It should be fixed, now.

**Comment by acos0874** [comment]
All good thank you. Would it be possible to also add balance tracking?
